checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0"?>
```

Comment:
if you're going to move files like this do it in another PR please,
but all of our other repos have it under `config` to not clutter the root directory:
https://github.com/hmcts/draft-store/tree/master/config

Reply:
I did not move anything, I just merged your changes. By the way, can you please make a PR instead of making changes to this branch?

Reply:
I could have done that but I just merged master to help you out...

Reply:
i.e. you haven't merged properly after pulling my merge